src/algebra/monoid_algebra.lean

```diff
@@ -4,6 +4,11 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Yury G. Kudryashov, Scott Morrison
 -/
 import algebra.algebra.basic
+import init.meta.interactive
+import init.meta.converter
+
+-- TODO: can we avoid needing this?
+meta def conv.interactive.exact := tactic.interactive.exact
```

Comment:
@digama0 @gebner @robertylewis I think such a tactic makes sense right?
If so, I guess @eric-wieser can move it to an appropriate file and add a docstring.

Reply:
Probably it should be its own PR since various `conv`-related docs may need to be updated.

Reply:
I moved it to an appropriate file, let me know if you need more changes. I'd still prefer for someone to come up with a proof that doesn't need this tactic...

Reply:
Moved to #4615